docs/evaluation/simulate.rst

```diff
@@ -45,3 +40,181 @@ NumPy or SciPy to evaluate noise, such as ``numpy.random.standard_normal``.
    >>> from numpy.random import standard_normal, normal
    >>> yconst = ymdl + standard_normal(ymdl.shape) * 10
    >>> ydata = ymdl + normal(scale=np.sqrt(ymdl))
+
+
+.. _data_pha_fake:
+
+X-ray data (`~sherpa.astro.data.DataPHA`)
+=========================================
+
+In principle, the same steps apply when simulating :term:`PHA` data
+(`~sherpa.astro.data.DataPHA` objects), however, the mechanics are a
+little more complicated because we need to account for the
+instrumental response (:term:`ARF` and :term:`RMF`) and possibly also
+the background, which may contribute to the source spectrum that we
+want to simulate.  Readers not interested in X-ray data analysis may
+want to skip this section.
+
+Sherpa offers a dedicated function `sherpa.astro.io.fake_pha` for
```

Comment:
```suggestion
Sherpa offers a dedicated function :py:func:`sherpa.astro.io.fake_pha` for
```

I think it's `:py:func:` - you'd have to `python setup.py build_sphinx` and then check